src/hooks/searchParams.tsx

```diff
@@ -0,0 +1,31 @@
+import { useSearchParams } from 'react-router-dom';
```

Comment:
Maybe for later: should this file go in ui?

Reply:
I don't think it's a module that has cross-cutting concerns. IMO the search params here are specific to the auth front-end, so they should stay here.